### ISSUE TITLE
DevDependencies inside Dependencies

### ISSUE THREAD

**Issue by sidharthv96** [ryansolid/create-solid#2]
Hi,
I created the typescript starter project and saw that things like typescript and @types are inside the dependencies inside package.json.
Is this by choice as solid is only supposed to generate the final files? I saw similar organisation inside this repo too.

I can create a PR to move stuff to devDependencies so that the project follows the universal standard. It will also allow people who are starting out with JS ecosystem to have a consistent feel.

**Comment by ryansolid** [comment]
Yeah I I just copied CRA(Create React App) and they put everything in a single dependencies array and added them. This project is just a fork of the React version of it.

These repos themselves(solid-scripts/create-solid) require the packages to run so they are dependencies. Since even if you are running them as dev-dependencies in a different project they need their own dependencies.

However, I agree the final outputted template doesn't require anything to be a dependency. It is a weird disconnect though as I have found that the majority starters still mark certain things as dependencies(like the library itself.. in this case `solid-js`) and certain things(like typescript, jest) as devDependencies. Even though they technically could all be devDependencies. I'm not clear why Create React App made these decisions but it seems like it would be reasonable to just make these dev deps.
